### DISCUSSION TITLE
Organization governance

### DISCUSSION THREAD

**Discussion by stafyniaksacha** [strapi-community/.github#4] [Q&A]
Let's discuss how we can manage this organization by the long term 

> https://www.redhat.com/en/blog/understanding-open-source-governance-models

**Comment by derrickmehaffy** [comment]
This one might help too: https://opensource.guide/leadership-and-governance/

**Comment by derrickmehaffy** [comment]
Ideally main questions to answer here are:

- What model does the community want to adopt?
- Depending on model chosen how should the first set of "leaders"/"council"/"board" be selected?
- In the future how will these "leaders"/"council"/"board" be selected?
- How should membership be handled and approved?
- What are requirements for membership? Rules? ect
